integration/js/pages/AppPage.js

```diff
@@ -279,20 +279,23 @@ class AppPage {
     return await this.driver.findElement(elements.failedMeetingFlow).isDisplayed();
   }
 
-  async rosterCheck(numberOfParticipants) {
+  async rosterCheck(numberOfParticipants, checkCount = 10, waitTimeMs = 100) {
     let i = 0;
-    let timeout = 10;
-    while (i < timeout) {
+    let start = performance.now();
+    while (i < checkCount) {
       try {
         const participantCountOnRoster = await this.getNumberOfParticipantsOnRoster();
         if (participantCountOnRoster === numberOfParticipants) {
+          let end = performance.now();
+          console.log("Roster check completed successfully in : %d secs", (start - end)/1000);
           return true;
         }
       } catch (err) {
       }
-      await TestUtils.waitAround(10);
+      await TestUtils.waitAround(waitTimeMs);
       i++;
     }
+    console.log("Roster check failed in : %d secs", (start - end)/1000);
```

Comment:
So if roster check fail, where do we populate end variable here?

Reply:
oh yeah fixing it